unstructured/chunking/title.py

```diff
@@ -115,22 +120,37 @@ def chunk_by_title(
         metadata = first_element.metadata
         start_char = 0
         for element in section:
+            # -- concatenate all element text in section into `text` --
             if isinstance(element, Text):
+                # -- add a blank line between "squashed" elements --
                 text += "\n\n" if text else ""
                 start_char = len(text)
                 text += element.text
+
+            # -- "chunk" metadata should include union of list-items in all its elements --
             for attr, value in vars(element.metadata).items():
                 if isinstance(value, list):
+                    value = cast(List[Any], value)
+                    # -- get existing (list) value from chunk_metadata --
                     _value = getattr(metadata, attr, []) or []
-
-                    if attr == "regex_metadata":
-                        for item in value:
-                            item["start"] += start_char
-                            item["end"] += start_char
-
+                    # TODO: this mutates the original, work on a copy instead.
```

Comment:
please create a github issue for this

Reply:
This is actually fixed in the next PR (which removes this comment), but I can take this out if you want.